CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- compas.utilities.maps.geometric_key2 renamed to geometric_key_xy
```

Comment:
This goes under the `Changed` headline, not `Added`

Reply:
Sorry, fixed!